src/graphql/resolvers/abilityScoreResolver.ts

```diff
@@ -1,15 +1,22 @@
 import SkillModel from '../../models/skill/index.js';
 import { coalesceFilters, getMongoSortDirection, resolveContainsStringFilter } from './common.js';
+import { AbilityScore } from '../../models/abilityScore/types';
+
+type SortParams = Record<string, 1 | -1>;
+type Args = {
+  name?: string;
+  order_direction?: string;
+};
 
 const AbilityScoreResolver = {
-  skills: async (abilityScore, args) => {
-    const filters = [{ index: { $in: abilityScore.skills.map(s => s.index) } }];
+  skills: async (abilityScore: AbilityScore, args: Args) => {
+    const filters: any[] = [{ index: { $in: abilityScore.skills?.map(s => s.index) } }];
```

Comment:
Is `abilityScore.skills` potentially undefined?

Reply:
No, but it can be an empty array. I've adjusted accordingly.